### ISSUE TITLE
Twitch : Register Application

### ISSUE THREAD

**Issue by RamblingGeekUK** [clarkio/vscode-twitch-highlighter#45]
![image](https://user-images.githubusercontent.com/7108949/52170952-d6209200-274b-11e9-926b-cbe5a4a446b9.png)

Should this be set to localhost?  Does it need a port?

**Comment by FletcherCodes** [comment]
You can just leave it at localhost if you're just using it for the Code Highlighter.

**Comment by FletcherCodes** [comment]
Have you gone through the steps on the README?

**Comment by RamblingGeekUK** [comment]
Yeah ... Twice... without the Channel name, it connects but doesn't work, with the channel name it doesn't connect and the bot never posts to the chat to say its connected. :-)

**Comment by RamblingGeekUK** [comment]
yeah very strange.  Happy to stream it...   if that helps

**Comment by clarkio** [comment]
I'm working on updating the instructions and how people should set up the twitch connection part. I'll have something out soon.

The quick answer is you don't need to create an app and therefore don't need a client id or password. Instead just generate an OAuth token with either your own account or a separate account here http://www.twitchapps.com/tmi. Set that as your password and you should be good to go.

I'm running through the steps a few times from a clean slate to double check but let me know if you all run into issue with that as well.

**Comment by RamblingGeekUK** [comment]
Awesome.  OK. Will give it try.  Thanks for the update.

**Comment by RamblingGeekUK** [comment]
Ok. So I have tried this.  I removed Client ID from VS Code and used the OAuth Password with and without the "oauth" and it prompts for twitch client ID which I think your saying is not required. 

![clarkio-th](https://user-images.githubusercontent.com/7108949/52378803-949c2980-2a60-11e9-8fc8-c496f15ff263.gif)

**Comment by clarkio** [comment]
Yea you can put anything you want there for now so just put something like "1" to make it happy and then move on to the password. I'm removing that in the PR I shared in Discord which I plan to release once tested/validated.
